sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_generated/_operations_mixin.py

```diff
@@ -54,13 +54,11 @@ def analyze_status(
         :type skip: int
         :keyword callable cls: A custom type or function that will be passed the direct response
         :return: AnalyzeJobState, or the result of cls(response)
-        :rtype: ~azure.ai.textanalytics.v3_1_preview_3.models.AnalyzeJobState
+        :rtype: ~azure.ai.textanalytics.v3_1_preview_4.models.AnalyzeJobState
```

Comment:
can you add a changelog for this addition?

Reply:
Added.  Let me know if there's anything missing there.